core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -125,24 +125,28 @@ describe('Creator dashboard controller', () => {
     });
   }));
 
-  it('should get exploration url successfully', function() {
+  it('should get the correct exploration editor page URL corresponding to a' +
+    ' given exploration ID', function() {
     var explorationId = '1';
     expect(ctrl.getExplorationUrl(explorationId)).toBe(
       '/create/' + explorationId);
   });
 
-  it('should get collection url successfully', function() {
+  it('should get the correct collection editor page URL corresponding to a' +
+    ' given collection ID', function() {
     var collectionId = '1';
     expect(ctrl.getCollectionUrl(collectionId)).toBe(
       '/collection_editor/create/' + collectionId);
   });
 
-  it('should show username popover according to its length', function() {
-    expect(ctrl.showUsernamePopover('abcdefghijk')).toBe('mouseenter');
-    expect(ctrl.showUsernamePopover('abc')).toBe('none');
-  });
+  it('should show username popover event type according to its length',
```

Comment:
I'm slightly confused here... Can you explain this?

Reply:
This method is used on popover-trigger in the template. Depending on the username length, the popover will be triggered by different events.